File: python/7 kyu/292.py
# https://www.codewars.com/kata/5831c204a31721e2ae000294/train/python

# Passed

def swap(st):
    vowels = {'a', 'e', 'i', 'o', 'u'}
    result = str()
    for char in st:
        if char.lower() in vowels:
            char = char.upper()
            
        result += char
        
    return result       

output = swap("Codewars")
print(output)

File: nodejs/7 kyu/292.js
// https://www.codewars.com/kata/56576f82ab83ee8268000059/train/javascript

// Passed

function spacey(array) {
  let result = array.map((elem, idx) => array.slice(0, idx + 1).join(""));
  return result;
}

const output = spacey(["this", "cheese", "has", "no", "holes"]);
console.log(output);